sentry.properties

```diff
@@ -0,0 +1,5 @@
+defaults.url=https://sentry.io/
```

Comment:
**issue**: actually this is what is supposed to be a secret.

Reply:
I disabled this secret and created a new one. When we get back to this let's create a secret for it.